bin/test/i18n/sync/test_in.rb

```diff
@@ -0,0 +1,47 @@
+require_relative '../../test_helper'
+require_relative '../../../i18n/sync/in'
+
+class I18n::Sync::InTest < Minitest::Test
```

Comment:
do we use a mix of Minitest and ActiveSupport within the codebase?

Reply:
We use `ActionMailer::TestCase` in the Ruby On Rails framework applications like `dashboard`, but `bin` and `lib` are within the frameworkless ruby codebase.

Reply:
We use `ActionMailer::TestCase` in Ruby On Rails framework applications like `dashboard`, but `bin` and `lib` are within the frameworkless ruby codebase.